routes/api.js

```diff
@@ -1,24 +1,108 @@
 const express = require('express');
-
+const validate = require('express-validation');
+var validation = require('../validation/question.js');
+const url = require('url');
 const router = express.Router();
+const { Questions, Answers } = require('../db');
+const bcrypt = require('bcrypt');
+
+const hash = '$2b$10$PjqecawtIkC0tAPJhKjHGOH7N8KyZuhMNoQtP79fE.zGnoHnbjuxe';
+
+validate.options({
+  status: 422,
+  statusText: 'Unprocessable Entity',
+});
 
 // Adding of a question
-router.post('/questions', function(req, res) {
-  res.send({ question: req.body });
+router.post('/questions', validate(validation), function(req, res) {
+  const { password, question, subjectId, level, answers } = req.body;
+  bcrypt.compare(password, hash).then(function(pass) {
+    if (pass) {
+      Questions.create(
+        {
+          question,
```

Comment:
Validate arguments.
You can probably just use the req.body object here since the keys are the same.